### ISSUE TITLE
Which version of pytorch is used in this code？

### ISSUE THREAD

**Issue by QL111111** [j-a-lin/DFANet_PyTorch#1]

**Comment by jandylin** [comment]
We used pyTorch v1.1! For some unprovided architecture details, we made our own appropriate choices (e.g. concatenation operations).

**Comment by yellowYuga** [comment]
Did u get the result in paper?

**Comment by jandylin** [comment]
We obtained about 64% mIoU on the training data (without ASPP).

**Comment by sonack** [comment]
> We obtained about 64% mIoU on the training data (without ASPP).

How about the test set's performance ?

**Comment by sonack** [comment]
> We obtained about 64% mIoU on the training data (without ASPP).

How about the test set's performance ?

**Comment by yellowYuga** [comment]
> We obtained about 64% mIoU on the training data (without ASPP).

Training data？How about on test set？
